package.json

```diff
@@ -20,6 +20,13 @@
     "build:server": "(cd client && npm run build:server --silent)",
     "build:dev:client": "(cd client && npm run build:dev:client --silent)",
     "build:dev:server": "(cd client && npm run build:dev:server --silent)",
+    "prebuild:production": "rake react_on_rails:locale",
+    "prebuild:production:client": "rake react_on_rails:locale",
+    "prebuild:production:server": "rake react_on_rails:locale",
+    "prebuild:client": "rake react_on_rails:locale",
+    "prebuild:server": "rake react_on_rails:locale",
+    "prebuild:dev:client": "rake react_on_rails:locale",
+    "prebuild:dev:server": "rake react_on_rails:locale",
```

Comment:
https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/.travis.yml#L32

If rake task is moved to `test.rb`, the above line will be executed before rake task. 
npm build will fails becuase there is no js locale files.

As to development.rb, I put rake task in procfile.dev.